### DISCUSSION TITLE
Stop Scan

### DISCUSSION THREAD

**Discussion by Nionor** [photoview/photoview#421] [Q&A and Troubleshooting] [answered]
Is there a way to stop the scanning process, I couldn't find anything in the UI but maybe from shell?
Or is it ok to just shutdown the container and restart the scanning.

**Comment by viktorstrate** [comment, marked as answer]
There is no way to stop the scanner yet, although there's an issue open for it here #31.
But it is safe to stop the server while it's scanning, when the scanner is started again later it will just continue where it left of.

**Reply by Nionor** [reply]
Ok, good to know. Thanks!
